PyLaTeXtables/templates/eoc.tex

```diff
@@ -4,7 +4,12 @@
 
 ((* set tabular_columns = index_column_types * index_columns + 'r' * ( data_columns % 2 ) + 'rc' * ( data_columns // 2 ) *))
 ((* set np_digits = 2 *))
-((* set header_is_hierarchical = df.columns.format(sparsify=True) != df.columns.format(sparsify=False) *))
+((* if header_is_hierarchical *))
+   % do something for multi-index
```

Comment:
This does not make sense. The file is an actual template, not an example for users.